### DISCUSSION TITLE
AWS Support

### DISCUSSION THREAD

**Discussion by mickdelaney** [OrchardCMS/OrchardCore#8508] [Q&A] [answered]
Hi Everyone,

We’re on AWS, and are hoping to select orchard as the centre piece to a new feature set for our platform. 
I’m trying to get an understanding of what we’ll need to do to get orchard working on AWS. We use kubernetes also. 
Any help much appreciated. 

regards

**Comment by jptissot** [comment, marked as answer]
We have been using OC on kubernetes. The chart does not currently support horizontal scaling. Hope this helps :)

Code repository: https://github.com/StatCan/StatCan.OrchardCore 
Helm Chart: https://github.com/StatCan/charts/tree/master/stable/orchard-cms
